app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -278,7 +281,7 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
             {this.getStatusIcon(OPEN_APP)}
           </div>
           <div className={styles.ledgerStageText}>
-            <FormattedHTMLMessage id="auth.ledger.navigateToNeoApp" />
+            Navigate to the <strong>NEO3 app</strong> on your device
```

Comment:
Now this component always says NEO3 app 😞

Reply:
Fixed with https://github.com/CityOfZion/neon-wallet/commit/e2a3773d150ffd8fff635fb38c4a17f77e334d47